spec/javascripts/Common/components/ToolbarSearch.spec.tsx

```diff
@@ -15,17 +17,51 @@ it('should render itself', () => {
   expect(wrapper.exists()).toEqual(true)
 })
 
-it('should have a placeholder', () => {
+it('can have a placeholder', () => {
   const placeholder = 'Find something'
   const wrapper = mountWrapper({ placeholder })
   expect(wrapper.exists(`input[placeholder="${placeholder}"]`)).toEqual(true)
 })
```

Comment:
Since it's now InputSearch's responsibility, we can remove this test. It doesn't harm but it's also useless.